### ISSUE TITLE
Clients View in Dashboard: Request failed with status code 500

### ISSUE THREAD

**Issue by PelleRavn** [emqx/emqx#3248]
#### Environment

- OS: Ubuntu 16.04
- Erlang/OTP: Don't know. Using Docker with emqx/emqx:latest
- EMQ: 4.0.2

#### Description

I have started a EMQ X cluster up with 3 nodes. And it have been running fine for a few days.
But now, whenever I go to the Dashboard menu > Monitoring > Clients, and select a specific client, I get a red bar from top, saying `Request failed with status code 500`.

Haven't changed the configuration, so I don't know what should have changed from my side.
Restarting the services doesn't help. 

If I look at the log from the Docker container, I get the following stack trace:

```
2020-02-12 14:07:51.047 [error] GET /api/v4/clients/34e8e66606a149958ea36e96335c56ea error: badarg, stacktrace:
[{erlang,'++',                                                                
         [#{},                                                                
          [#{recv_pkt => 182721,recv_oct => 58480249,is_bridge => false,      
             max_inflight => 32,connected_at => <<"2020-02-12 10:26:47">>,    
             send_pkt => 183594,zone => external,connected => true,           
             recv_cnt => 83519,mqueue_len => 0,clean_start => false,          
             proto_name => <<"MQTT">>,port => 36450,max_subscriptions => 0,   
             reductions => 159531715,                                         
             clientid => <<"34e8e66606a149958ea36e96335c56ea">>,              
             created_at => <<"2020-02-12 10:26:47">>,send_cnt => 183594,      
             mqueue_dropped => 0,max_mqueue => 1000,send_oct => 732630,       
             subscriptions_cnt => 0,send_msg => 0,node => 'emqx@10.42.1.82',  
             keepalive => 15,heap_size => 4185,max_awaiting_rel => 100,       
             ip_address => <<"10.42.1.90">>,inflight => 0,proto_ver => 4,     
             mailbox_len => 0,username => undefined,awaiting_rel => 0,        
             recv_msg => 91360,expiry_interval => 7200}|                      
           #{}]],                                  
         []},                                      
 {lists,append,1,[{file,"lists.erl"},{line,127}]}, 
 {emqx_mgmt_api_clients,lookup,2,                                                                              
                        [{file,"/emqx_rel/_build/emqx/lib/emqx_management/src/emqx_mgmt_api_clients.erl"},     
                         {line,111}]},                                                                         
 {minirest_handler,dispatch,2,                                                             
                   [{file,"/emqx_rel/_build/emqx/lib/minirest/src/minirest_handler.erl"},  
                    {line,84}]},                                                           
 {minirest,handle_request,2,                                             
           [{file,"/emqx_rel/_build/emqx/lib/minirest/src/minirest.erl"},
            {line,94}]},                                                 
 {minirest,init,2,                                                       
           [{file,"/emqx_rel/_build/emqx/lib/minirest/src/minirest.erl"},
            {line,86}]},                                                 
 {cowboy_handler,execute,2,                                                                
                 [{file,"/emqx_rel/_build/emqx/lib/cowboy/src/cowboy_handler.erl"},        
                  {line,41}]},                                                             
 {cowboy_stream_h,execute,3,                                                               
                  [{file,"/emqx_rel/_build/emqx/lib/cowboy/src/cowboy_stream_h.erl"},      
                   {line,320}]}]
```

**Comment by tigercl** [comment]
@PelleRavn Sorry for late reply, we confirm this is a bug, and we will fix it in next release, sorry for causing you trouble.

**Comment by tigercl** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix issue#](https://trello.com/c/frTvEqT0/84-fix-issue)

**Comment by HJianBo** [comment]
Hi guys, It has fixed at v4.0.3
